library/src/main/java/com/kizitonwose/calendarview/ui/CalendarAdapter.kt

```diff
@@ -281,6 +283,11 @@ internal class CalendarAdapter(
 
     fun findLastVisibleDay(): CalendarDay? = findVisibleDay(false)
 
+    private fun View.getMarginHeight(): Int {
```

Comment:
I think naming this `getVerticalMargins()` will be more accurate. Could also be a computed property(no strong opinion on this one though).

Also, would be nice to have it in the `Extensions` file in the utils package so it can be reused.

Reply:
sounds great, updating the patch!